tests/conftest.py

```diff
@@ -97,8 +97,9 @@ def _source_image(tmpdir_factory):
 
     if not pathlib.Path(src_image_file_path).is_file():
         print("\nDownloading Source Image")
-        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata"
-                         "/common/source_image.png")
+        r = requests.get(
+            "https://tiatoolbox.dcs.warwick.ac.uk/testdata" "/common/source_image.png"
```

Comment:
why splitting the path?

Reply:
Ok, I have checked and black v21 formats it like this:

```python
r = requests.get(
            "https://tiatoolbox.dcs.warwick.ac.uk/testdata" "/common/source_image.png"
)
```

Reply:
Fixed this now